docfx/pulumi-template/ManagedReference.common.js

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.

```

Comment:
It's a bit of a shame we have to copy in all these files - are they not available somewhere we can reference instead?  If vendoring them like this recommended?

Assuming we haven't made any changes - perhaps add a README in the folder with details about where this all comes from?

Reply:
Great point! I found out that we can do overrides with themes, so I removed all files that were untouched and left the ones I changed.